README.md

```diff
@@ -1,73 +1,24 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# README
 
-# Project Name
+This README would normally document whatever steps are necessary to get the
+application up and running.
 
-> Description the project.
+Things you may want to cover:
 
+* Ruby version
 
-## Built With
+* System dependencies
 
-- Major languages
-- Frameworks
-- Technologies used
+* Configuration
 
-## Live Demo (if available)
+* Database creation
 
-[Live Demo Link](https://livedemo.com)
+* Database initialization
 
+* How to run the test suite
 
-## Getting Started
+* Services (job queues, cache servers, search engines, etc.)
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+* Deployment instructions
 
-
-To get a local copy up and running follow these simple example steps.
-
-### Prerequisites
-
-### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
-
-## Authors
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
-
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
+* ...
```

Comment:
- [ ] Please make sure to use a proper readme template (like the one provided by Microverse), and make sure that all section is edited appropriately to better suit your project.